components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2663,21 +2663,25 @@ where
                 && !is_initialized_peer
                 && msg_type == target_msg_type
         };
+        #[cfg(feature = "failpoints")]
```

Comment:
fp_enable is cfg under failpoints, so the the code use it should cfg under failpoints

Reply:
`fail_point` macro expand to noop if `failpoints` is disabled, so there seems no need to add this explicit cfg attr, do you meet a compiling error here?

Reply:
yes, you may run tests in in_memory_engine/tests/failpoints
such as `cargo test --package in_memory_engine --test failpoints --features failpoints -- test_memory_engine::test_gc_worker --exact --nocapture`

Reply:
So this is a problem of in_memory_engine rather that raftstore, you should enable `"raftstore/failpoints"` in in_memory_engine's failpoints feature(in in_memory_engine/Cargo.toml)